lib/features/base/interactors_bindings.dart

```diff
@@ -0,0 +1,24 @@
+
+import 'package:get/get_instance/src/bindings_interface.dart';
+
+abstract class InteractorsBindings extends Bindings {
```

Comment:
Why we need this class?

Reply:
I want to separate the injecting of classes that handle the data and domain layers